vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.h

```diff
@@ -577,6 +582,27 @@ class LedgerImpl : public ledger::Ledger,
       int32_t current_country_code,
       ledger::GetAddressesCallback callback);
 
+  void OnGetAllTransactions(
+      ledger::mojom::AllTransactionsPtr transactions,
+      uint32_t record,
+      ledger::ACTIVITY_MONTH month,
+      uint32_t year,
+      double current_balance,
+      ledger::MonthlyStatementCallback callback);
```

Comment:
not sure if MonthlyStatement is something that ledger should care about. It's just returning some range of Transactions. It also seems a bit strange to me that the method is called `GetAllTransactions`/`OnGetAllTransactions` when it isn't getting all the transactions. I'm not really sure what the method should be called though because I'm not following exactly what we are returning since `AllTransactions` holds different types of data